tests/integration/modules/test_cp.py

```diff
@@ -64,6 +64,7 @@ def test_get_file_templated_paths(self):
             self.assertIn('Gromit', data)
             self.assertNotIn('bacon', data)
 
+    @skipIf(__salt_system_encoding__ == 'US-ASCII', 'Requires a unicode locale, i.e. en_US.UTF-8')
```

Comment:
We can skip, but we shouldn't. We should be able to handle gzipped files whether we're on a system with a unicode locale set or not.